tiledb/sm/query/query_buffer.h

```diff
@@ -201,6 +201,41 @@ class QueryBuffer {
    * attributes.
    */
   ValidityVector validity_vector_;
+
+  /* ********************************* */
+  /*           PUBLIC METHODS         */
+  /* ********************************* */
+
+  Status set_data_buffer(void* data_buffer, uint64_t* size) {
+    buffer_ = data_buffer;
+    buffer_size_ = size;
+    original_buffer_size_ = *size;
+
+    return Status::Ok();
+  }
+
+  Status set_data_var_buffer(void* data_var_buffer, uint64_t* size) {
```

Comment:
Remove this, it is subsumed by `set_offsets_buffer` below.

Reply:
Let's ignore this for now, I see that the functions are not equivalent. Thanks @bekadavis9 for clarifying offline.